.storybook/custom-addons/provider/index.js

```diff
@@ -41,15 +44,16 @@ function ProviderUpdater(props) {
     };
   }, []);
 
-  let statusMap = {
-    'positive': "Released as rc",
-    'notice': "In alpha",
-    'negative': "Under construction"
-  }
-
   return (
     <Provider theme={theme} colorScheme={colorScheme} scale={scaleValue} locale={localeValue} toastPlacement={toastPositionValue}>
-      <div style={{position: 'absolute', paddingTop: '20px', paddingLeft: '20px', paddingRight: '20px'}}><div style={{fontSize: '18px', paddingLeft: '10px', paddingRight: '10px'}}><strong>Status</strong></div><StatusLight variant={props.options.status || 'negative'}>{statusMap[props.options.status || 'negative']}</StatusLight></div>
+      <div style={{position: 'absolute', paddingTop: '20px', paddingLeft: '20px', paddingRight: '20px'}}>
+        {props.context.parameters.note && (<DialogTrigger type="popover">
+          <ActionButton isQuiet>Note</ActionButton>
+          <Dialog>
+            <Content><Text>{props.context.parameters.note}</Text></Content>
+          </Dialog>
+        </DialogTrigger>)}
+      </div>
```

Comment:
IMO, it would be nice if this was a tooltip so it could show on hover, especially since quiet ActionButtons look like normal text (that or maybe if there was an info icon to go along with it). Don't need to change your pull, just an observation I had

Reply:
The original idea was to use a Tooltip, but the text like the one on Tabs was excessive for a tooltip which is why this design suggestion using a Dialog. That experiment did find an issue with Tooltip related to the carrot and also was why I was asking about Tooltip width

Reply:
Ahhh, that makes sense